### PR TITLE
Doc-213-Ubuntu-v20-cc-eal-disabled

### PR DESCRIPTION
**Pull request by Siboney1**
noted cc-eal is disabled in v20

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/804#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @Siboney1. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/06-integrations/00-ubuntu.md [hunk 1]
```diff
@@ -166,21 +168,29 @@ For more information see the [Ubuntu Advantage for Infrastructure](https://ubunt
...
+|||||
+| **FIPS** | enable <br /> <br /> <br /> <br /> disable | Federal Information Processing Standards (FIPS) 140 validated cryptography for  <br /> Linux workloads on Ubuntu. Install, configure, and enable <br /> FIPS 140 certified modules.  <br /><br /> Do not have FIPS 140 validated cryptography for Linux workloads on Ubuntu.|
+|||||
+| **FIPS-updates** | enable <br /> <br /> <br /> disable | The option enables the FIPS-UPDATES. It installs the updated FIPS modules sets <br />it in FIPS mode. <br /><br /> Do not have FIPS 140 validated cryptography for Linux workloads on Ubuntu.|
+|||||
+| **CC-EAL** | enable <br /> <br /> <br /> <br /> <br />  disable | After the completion of a Common Criteria (CC) security evaluation, a grade is given <br /> indicating the level the system was tested. Common criteria evaluated configuration is <br /> currently available for Ubuntu 16.04.4 LTS (Server) and Ubuntu 18.04.4 LTS (Server). <br /> The option shows as disabled for Ubuntu 20 & 20+ as per this documentation. <br /> <br /> Do not make enable and install the CC artifacts. |
```
Comment (truegilb) [inline content/docs/06-integrations/00-ubuntu.md:191]: "as per this documentation" - is there a link on Canonical website to reference?
"Do not make enable and install the CC artifacts." - slight word change of "make enable"?

**Review by truegilb — CHANGES_REQUESTED** [review]

**Review by resmaa — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/804#pullrequestreview-1073444671" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/804#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by resmaa — APPROVED** [review]
apprvd

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/804#pullrequestreview-1073637513" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/804#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by Siboney1 — COMMENTED** [review]
fixed

File: content/docs/06-integrations/00-ubuntu.md [hunk 2]
```diff
@@ -141,13 +142,14 @@ For more information see the [Ubuntu Advantage for Infrastructure](https://ubunt
...
 
+
 1. Palette allows you to include the Ubuntu Advantage service in the **Profile Layers** section, when you create a new cluster profile. 
 
 
 2. Give the new Pack a **Name**, **Version number**, **Description**, **Type**, and **Tags** and click the **Next** button.
 
 
-3. Choose the cloud provider as the **Infrastructure provider** and click the **Next** button.
+3. Choose the cloud provider as the **Infrastructure provider** and click the **Next**.
```
Comment (truegilb) [inline content/docs/06-integrations/00-ubuntu.md:152]: and click **Next** (no the)

File: content/docs/06-integrations/00-ubuntu.md [hunk 3]
```diff
@@ -125,7 +126,7 @@ Spectro Golden images include most of the hardening standards recommended by CIS
 
 # Ubuntu Advantage
 
-Canonical Ubuntu Advantage extends your infrastructure's security, certified compliance, and 24x7 support. With Palette, enable the UA services when modifying the Pack Values. See below for steps on how to modify the Preset options.
+Ubuntu Advantage services extend your infrastructure's security, compliance, and productivity requirements. With Palette, enable the UA services when modifying the Pack Values. See below for steps on how to modify the Preset options.
```
Comment (truegilb) [inline content/docs/06-integrations/00-ubuntu.md:129]: Is there better way to represent "With Palette, enable the UA services when modifying the Pack Values."

**Review by truegilb — CHANGES_REQUESTED** [review]
Small changes please. Overall looks good.

**Review by truegilb — APPROVED** [review]
Looks good.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/804#pullrequestreview-1073637513" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/804#" title="Author self-approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/804#pullrequestreview-1099899018" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by truegilb — APPROVED** [review]
Looks good.

**Review by truegilb — COMMENTED** [review]
Looks good.

**Pull request merged**